src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/install-android-apps-gradle-android-studio.mdx

```diff
@@ -33,115 +33,188 @@ redirects:
 
-  <CollapserGroup>
-    <Collapser
-      id="project-level"
-      title="Project level build.gradle file:"
-    >
-      In this example, AGENT_VERSION represents your agent version number. See the [agent release notes](/docs/release-notes/mobile-release-notes/android-release-notes), and use the latest version.
+```java
+repositories {
+  mavenCentral()
+}
 
-      ```
-      buildscript {
-        repositories {
-          mavenCentral()
-        }
+apply plugin: 'android'
+apply plugin: 'newrelic'
 
-        dependencies {
-          classpath "com.newrelic.agent.android:agent-gradle-plugin:<a href="/docs/release-notes/mobile-release-notes/android-release-notes">AGENT_VERSION</a>"
-        }
-      }
-      ```
-    </Collapser>
-
-    <Collapser
-      id="app-level"
-      title="App level build.gradle file:"
-    >
-      In this example, AGENT_VERSION represents your agent version number. See the [agent release notes](/docs/release-notes/mobile-release-notes/android-release-notes), and use the latest version.
-
-      ```
-      repositories {
-        mavenCentral()
-      }
+dependencies {
+  implementation 'com.newrelic.agent.android:android-agent:<a href="/docs/release-notes/mobile-release-notes/android-release-notes">AGENT_VERSION</a>'
+}
+```
+
+In both scripts, replace `<AGENT_VERSION>` with the most recent agent version. See the [agent release notes](/docs/release-notes/mobile-release-notes/android-release-notes) to stay up to date. You may choose to configure one or both depending on your monitoring needs. 
+
+    </Step>
+    <Step>
+        ### Set app permissions 
+
+  Set app permissions to make requests to `INTERNET` and `ACCESS_NETWORK_STATE`. Add this code snippet to your `AndroidManifest.xml` file:
+
+```
+  <uses-permission android:name="android.permission.INTERNET" />
+  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+```
+
+        
+    </Step>
+    <Step>
+        ### Start the Android agent on app start up
+
+    In your **Default Activity**, import the `NewRelic` class. Your **Default Activity** is defined in your **Manifest**. 
+
+```
+<uses-permission android:name="android.permission.INTERNET" />
+<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+```
+
+To  initialize mobile monitoring, add the following script in your `onCreate()` method:
+
+```
+import com.newrelic.agent.android.NewRelic;
+```
+
+    </Step>
+</Steps>
+
+You may prefer completing the configuration process for a new mobile app later. To do that, go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile**, then select **See instructions** next to your mobile app name.
+
+If you want to upgrade an existing Android installation, then go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > (select an app) > Settings > Installation**.
+
+## Set up Gradle DSL with Gradle Plugin Portal [#gradle-dsl]
```

Comment:
@cthomas-newrelic  Based on our conversation, it seems like this is an additional step that only some users will take. I've broken this out of the main install/set up flow. 

- The original PR nested this as an intermediary step. Do users who want this feature need to set it up at a specific point in install, or can they go back after they're set up and add this plugin?
- If the first, we can leave as is; if the second, I'll want to move this content back in a clever way